getting_started/intel_debug/suggestions.rst

```diff
@@ -294,10 +294,31 @@ driver:
    #define SOC_ES8336_HEADSET_MIC1		BIT(8)
 
 
-The default quirk value for the platform can be read from
+The default and actual quirk values for the platform can be got
+from the dmesg log file (/var/log/dmesg) in the line like:
+
+.. code-block:: txt
+
+    ...
+    sof-essx8336 sof-essx8336: quirk mask 0x1a0
+    ...
+
+for the unchanged mask, or:
+
+.. code-block:: txt
```

Comment:
This is invalid for some reason:

https://github.com/thesofproject/sof-docs/actions/runs/6297068973/job/17095424648?pr=477

```
/home/runner/work/sof-docs/sof-docs/getting_started/intel_debug/suggestions.rst:300: WARNING: Pygments lexer name 'txt' is not known
/home/runner/work/sof-docs/sof-docs/getting_started/intel_debug/suggestions.rst:308: WARNING: Pygments lexer name 'txt' is not known

```

Reply:
the code-block fixed to `none`.